protobufs/livekit_sip.proto

```diff
@@ -206,6 +213,27 @@ message SIPOutboundTrunkInfo {
   // May be empty to have no authentication.
   string auth_username = 7;
   string auth_password = 8;
+
+  // Include these SIP X-* headers in INVITE request.
+  map<string, string> headers = 9;
+  // Map SIP X-* headers from 200 OK to SIP participant attributes.
+  map<string, string> headers_to_attributes = 10;
+}
+
+message GetSIPInboundTrunkRequest {
```

Comment:
It's unclear to me what TrunkRequests and TrunkResponses are?

Reply:
You mean it's unclear why they are here? I just noticed that our OSS API for managing Trunks supports listing trunks, but cannot get a Trunk by ID. So I'm adding it here.

Reply:
Never mind, I misread here.